### DISCUSSION TITLE
[Bug] Published again and again but not updated on the web?

### DISCUSSION THREAD

**Discussion by phankhanhhung** [webstudio-is/webstudio-community#69] [Q&A]
Dear Webstudio.is Team,

I'm encountering an ongoing problem with Webstudio.is where despite publishing my site multiple times, it only displays the previous version, although the tip has shown on the icon is just published in x minutes. I need urgent assistance in resolving this issue. If a solution is not readily available, I am considering self-deployment or using Vercel. Your prompt attention to this matter is appreciated.

Thank you,

**Comment by kof** [comment]
Hey @phankhanhhung a couple of things:

1. We are happy either way if you deploy on webstudio or vercel
2. We don't guarantee prompt attention, so this expectation needs to change, you are on a free plan, though we will always try to help as much as we can.
3. When you submit a request for help you need to provide a share link so we can even look, without it there is nothing we can do
